concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/contractexample/parameters/UpgradeParams.java

```diff
@@ -0,0 +1,51 @@
+package com.concordium.sdk.examples.contractexample.parameters;
+
+import com.concordium.sdk.transactions.ReceiveName;
+import com.concordium.sdk.transactions.smartcontracts.Schema;
+import com.concordium.sdk.transactions.smartcontracts.SchemaParameter;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.Getter;
+import org.apache.commons.codec.binary.Hex;
+
+import java.io.IOException;
+
+/**
+ * Represents the parameter '<a href="https://github.com/Concordium/concordium-rust-smart-contracts/blob/main/examples/cis2-wccd/src/lib.rs#L150">UpgradeParams</a>' used in the cis2-wCCD contract
+ */
+@Getter
+public class UpgradeParams extends SchemaParameter {
+
+    private final String module;
```

Comment:
It is now a `ModuleReference`. To make it work i had to implement custom serialization of `Hash`, but this serialisation does the same as the already present `toString` method, i.e. returns the value as hex, so i believe this will be fine? All tets pass at least.

Reply:
Yes, it should just be the base16 encoded hash.